lib/semian.rb

```diff
@@ -162,7 +167,7 @@ def resources
     Semian.logger.info("Semian sysv semaphores are not supported on #{RUBY_PLATFORM} - all operations will no-op")
   end
 
-  unless Semian.semaphores_enabled?
+  if ENV['SEMIAN_SEMAPHORES_DISABLED']
```

Comment:
why this change? seems more reasonable to use the method instead of check the env var again